eng/Versions.props

```diff
@@ -10,40 +10,37 @@
     <UsingToolSymbolUploader>true</UsingToolSymbolUploader>
```

Comment:
Let's update `VersionPrefix` to `4.0.0` since this is Dev17.

Reply:
Updated this and the vsixmanifest